src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -6,7 +6,7 @@ import Linkify from "react-linkify";
 import { FlexBox, Button, Menu, Tooltip } from "../";
 import { componentDecorator, matchDecorator } from "../MessagingBubble/linkifyUtils";
 import Checkmark from "../Checkbox/CheckMark";
-import { MessagingTheme } from "src/utils/messaging";
+import { MessagingTheme } from "..//utils/messaging";
```

Comment:
nit: mayeb a typo here and at several other places? should be  `../` instead of `..//`

Reply:
OH wow derp...